backend/uclapi/dashboard/models.py

```diff
@@ -96,3 +101,43 @@ class APICall(models.Model):
 
     class Meta:
         _DATABASE = 'default'
+
+
+class Webhook(models.Model):
```

Comment:
Should we be placing Webhook logic in the dashboard models file? I feel like it's the same thing that you called me out on :P

Reply:
😜 
There's no separate app for webhooks (it's part of `dashboard`) and I think what I mentioned in the OAuth PR is oauth specific fields being part of the same table as `app`, but in this case webhook specific data is in its own table.

Reply:
No problem then  ;P